calico-cloud/network-policy/recommendations/learn-about-policy-recommendations.mdx

```diff
@@ -0,0 +1,145 @@
+---
+description: Learn more about policy recommendations.
+---
+
+# Learn more about policy recommendations
+
+## Big picture
+
+To use the [policy recommendation feature](policy-recommendations.mdx), you must have traffic between endpoints. In this tutorial we show you how recommendations are generated using flow logs in your cluster, for traffic to/from namespaces, network sets, private network IPs and public domains.
```

Comment:
```suggestion
 In this tutorial, we show you how recommendations are generated using flow logs in your cluster, for traffic to/from namespaces, network sets, private network IPs and public domains. Because the [policy recommendation feature](policy-recommendations.mdx), requires traffic between endpoints, a tutorial YAML is provided to create these resources. If your cluster is already generating policies, skip to the section [enable-policy-recommendations](#enable-policy-recommendations).
```